pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -180,6 +180,7 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 		return reconcile.Result{}, errors.Wrap(err, "failed to populate cluster info")
 	}
 	r.clusterInfo.Context = r.opManagerContext
+	r.clusterInfo.OwnerInfo = k8sutil.NewOwnerInfo(&cephCluster, r.scheme)
```

Comment:
@BlaineEXE https://github.com/rook/rook/pull/14079 is getting the ownerRef from the secret, why not get the owner for the ceph cluster instead of the secret?

Reply:
i didnt notice the above PR it as i have checked out 4.16 code :D